Source/WebCore/contentextensions/ContentExtensionsBackend.cpp

```diff
@@ -172,8 +172,11 @@ auto ContentExtensionsBackend::actionsForResourceLoad(const ResourceLoadInfo& re
     actionsVector.reserveInitialCapacity(m_contentExtensions.size());
     ASSERT(!(resourceLoadInfo.getResourceFlags() & ActionConditionMask));
     const ResourceFlags flags = resourceLoadInfo.getResourceFlags() | ActionConditionMask;
-    for (auto& contentExtension : m_contentExtensions.values())
+    for (auto& [identifier, contentExtension] : m_contentExtensions) {
+        if (shouldSkipRuleList && shouldSkipRuleList(identifier) == ShouldSkipRuleList::Yes)
```

Comment:
Don't do a null check inside the inner loop.  Make it always non-null.  You might even be able to use ScopedLambda

Reply:
Good point! I'll refactor to avoid these extra null checks (here, as well as the `loader` check below).